engines/dfc_provider/config/routes.rb

```diff
@@ -7,9 +7,18 @@
     resources :offers, only: [:show, :update]
     resources :supplied_products, only: [:create, :show, :update]
     resources :social_medias, only: [:show]
+    resources :sale_sessions, only: [:show]
+    resources :offers, only: [:show]
+    resources :orders, only: [:show] do
+      resources :order_lines, only: [:show]
+    end
```

Comment:
I'm guessing these have been added in order to build the URLs for the resource IDs. But so far this PR doesn't actually implement the endpoints. 
I'm wondering if we need to, or it's fine to just leave these un-implemented?

Reply:
Hm, I guess that your theory is right. But I don't think it's good to add URLs if they are not implemented. Instead, the ids should be `nil`. The export should then allocate random internal ids.

Reply:
Thanks, so I should be able to remove these routes, and set the ID to nil, for example:

```ruby
    def build_offer(sale)
      DataFoodConsortium::Connector::Offer.new(
        nil, # urls.enterprise_offer_url(sale.producer_id, sale.product_id),
        stockLimitation: nil,
      )
    end
```

Reply:
Yes, I think so.